Zend/tests/036.phpt

```diff
@@ -11,4 +11,4 @@ try {
 
 ?>
 --EXPECT--
-Illegal offset type
+Illegal array key type: Closure
```

Comment:
There is a compile-time similar error message, so I decided to also change the run-time ones as well.

Reply:
Also, leaving the `:` felt a little bit weird for me, but I'm ok to do so for consistency with other messages.

Reply:
No, not exactly. And the `Illegal array key of type Closure` format is the best; unfortunately, it didn't come to my mind :)

Speaking about which branch to target: I was also unsure which branch to use. I first thought that the changes won't be very offensive, but it turned out that they are, so I agree, let's instead go for PHP 8.1.